lightning-background-processor/src/lib.rs

```diff
@@ -19,7 +19,9 @@ extern crate core;
 #[cfg(not(feature = "std"))]
 extern crate alloc;
 
-#[macro_use] extern crate lightning;
+#[cfg(not(feature = "std"))]
+use lightning::{_log_warn as log_warn};
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F3o85xET02JdV8TQuwU%2Fgiphy.gif&token=goodcatch" data-gitmeme-token="goodcatch"><img src="https://media3.giphy.com/media/3o85xET02JdV8TQuwU/giphy.gif" title="Created by gitme.me with /goodcatch"/></a>
good catch!
yeah for this ill re-export the default/context logging module so that its visible externally